eng/scripts/Language-Settings.ps1

```diff
@@ -39,75 +38,26 @@ function Get-javascript-AdditionalValidationPackagesFromPackageSet {
     [Parameter(Mandatory = $true)]
     $AllPkgProps
   )
-  $additionalValidationPackages = @()
-
-  function isOther($fileName) {
-    $startsWithPrefixes = @(".config", ".devcontainer", ".github", ".scripts", ".vscode", "common", "design", "documentation", "eng", "samples")
-
-    foreach ($prefix in $startsWithPrefixes) {
-      if ($fileName.StartsWith($prefix)) {
-        return $true
-      }
-    }
-
-    return $false
-  }
-
-  $targetedFiles = $diffObj.ChangedFiles
-  if ($diff.DeletedFiles) {
-    if (-not $targetedFiles) {
-      $targetedFiles = @()
-    }
-    $targetedFiles += $diff.DeletedFiles
-  }
-
-  # The targetedFiles needs to filter out anything in the ExcludePaths
-  # otherwise it'll end up processing things below that it shouldn't be.
-  foreach ($excludePath in $diffObj.ExcludePaths) {
-    $targetedFiles = $targetedFiles | Where-Object { -not $_.StartsWith($excludePath) }
-  }
-
-  $changedServices = @()
-  foreach ($file in $targetedFiles) {
-    $pathComponents = $file -split "/"
-    # handle changes only in sdk/<service>/<file>/<extension>
-    if ($pathComponents.Length -eq 3 -and $pathComponents[0] -eq "sdk") {
-      $changedServices += $pathComponents[1]
-    }
-
-    # handle any changes under sdk/<file>.<extension> or in the root of
-    # the repository
-    if (($pathComponents.Length -eq 2 -and $pathComponents[0] -eq "sdk") -or
-        ($pathComponents.Length -eq 1)) {
-      $changedServices += "template"
-    }
-  }
-  $othersChanged = @()
-
-  if ($targetedFiles) {
-    $othersChanged = $targetedFiles | Where-Object { isOther $_ }
-  }
-  $changedServices = $changedServices | Get-Unique
-
-  if ($othersChanged) {
-    $additionalPackages = $ReducedDependencyLookup["core"] | ForEach-Object { $me = $_; $AllPkgProps | Where-Object { $_.Name -eq $me } | Select-Object -First 1 }
-    $additionalValidationPackages += $additionalPackages
-  }
+  $existingPackages = $LocatedPackages | ForEach-Object { $_.Name }
```

Comment:
```suggestion
  $existingPackages = $($LocatedPackages | ForEach-Object { $_.Name })
```
Not sure if necessary, but if it is a singular result this could screw up the `-notcontains` check below.